sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 ## Release History
 
-### 1.3.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+## 1.2.2 (2022-05-13)
+### Other Changes
+#### Dependency Updates
+- Upgraded `azure-communication-common` to 1.1.3
+- Upgraded `azure-communication-identity` to 1.1.9
```

Comment:
I hadn't noticed `azure-communication-identity` is a test dependency. We don't normally mention updates to those in our CHANGELOGs. By the way, `azure-core` should be mentioned here as it's widely used across the library.
```suggestion
- Upgraded `azure-communication-common` to 1.1.3
- Upgraded `azure-core` to 1.28.0
```